dali/operators/reader/loader/video/video_loader_decoder_gpu.cc

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) 2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "dali/operators/reader/loader/video/video_loader_decoder_gpu.h"
+
+namespace dali {
+void VideoSampleGpu::DecodeToOutput(uint8_t *output, cudaStream_t stream) {
+  video_file->SetCudaStream(stream);
```

Comment:
Done. This was due to the fact that `VideoSampleGpu` was a `struct` not a `class` and convention for structs is different. Since I expanded it with some functionality during this PR now it qualifies as `class` so I used `member_` convention.